### PR TITLE
warnings as error from true to false

### PR DESCRIPTION
**Pull request by emorikawa**
I simply set errors and warnings to true for now.
It looks like the warning is around 64-bit pointers from getting cast to 32 bit pointers at lines 92 and 111.

Everything appears to work despite the warnings.

```
..\src\keyboard-layout-observer-windows.cc(92): warning C4311: 'type cast': pointer truncation from 'HKL' to 'UINT' [C:\Code\N1\node_modules\
keyboard-layout\build\keyboard-layout-observer.vcxproj]
..\src\keyboard-layout-observer-windows.cc(92): warning C4302: 'type cast': truncation from 'HKL' to 'UINT' [C:\Code\N1\node_modules\keyboard
-layout\build\keyboard-layout-observer.vcxproj]
..\src\keyboard-layout-observer-windows.cc(111): warning C4311: 'type cast': pointer truncation from 'HKL' to 'UINT' [C:\Code\N1\node_modules
\keyboard-layout\build\keyboard-layout-observer.vcxproj]
..\src\keyboard-layout-observer-windows.cc(111): warning C4302: 'type cast': truncation from 'HKL' to 'UINT' [C:\Code\N1\node_modules\keyboar
d-layout\build\keyboard-layout-observer.vcxproj]
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 node_modules/
-lib/
```
Comment (thedaniel) [inline .gitignore]: why is this line removed?

**Comment by thomasjo** [comment]
Since this will no longer merge cleanly, and since there is no justification for changing the `WarnAsError` flag, I'm going to close this pull request. If you still feel strongly about changing the flag, please open a new pull request where you only change the flag, and provide a detailed description of why the change should be merged.

Thanks for the contribution! :bowing_man:
